File: perf/bench/sieve/sieve.go
package main

import "fmt"

const N = 2000000

func main() {
	flags := make([]bool, N)
	for i := range flags {
		flags[i] = true
	}
	var count int64 = 0
	for rep := 0; rep < 10; rep++ {
	count = 0
	for i := range flags { flags[i] = true }
	for i := 2; i < N; i++ {
		if flags[i] {
			count++
			for j := i + i; j < N; j += i {
				flags[j] = false
			}
		}
	}
	}
	fmt.Println(count)
}

File: perf/bench/sieve/sieve.swift
let N = 2000000
var flags = [Bool](repeating: true, count: N)
var count: Int64 = 0
for _ in 0..<10 {
    count = 0
    for i in 0..<N { flags[i] = true }
    var i = 2
    while i < N {
        if flags[i] {
            count += 1
            var j = i + i
            while j < N { flags[j] = false; j += i }
        }
        i += 1
    }
}
print(count)
